client/src/main/java/jp/co/soramitsu/iroha/java/debug/Account.java

```diff
@@ -1,18 +1,45 @@
 package jp.co.soramitsu.iroha.java.debug;
 
-import java.security.KeyPair;
+import java.security.*;
 import jp.co.soramitsu.crypto.ed25519.Ed25519Sha3;
+import jp.co.soramitsu.iroha.java.Utils;
 import lombok.Value;
+import net.i2p.crypto.eddsa.EdDSAPrivateKey;
+import net.i2p.crypto.eddsa.EdDSAPublicKey;
+import net.i2p.crypto.eddsa.spec.*;
+import static javax.xml.bind.DatatypeConverter.parseHexBinary;
+import static jp.co.soramitsu.crypto.ed25519.spec.EdDSANamedCurveTable.ED_25519;
 
 @Value
 public class Account {
 
-  private static Ed25519Sha3 crypto = new Ed25519Sha3();
+  private static Ed25519Sha3 edSha3Crypto = new Ed25519Sha3();
 
   private String id;
   private KeyPair keyPair;
 
+  public KeyPair getKeyPair() {
+    return keyPair;
+  }
+
   public static Account create(String id) {
-    return new Account(id, crypto.generateKeypair());
+    return new Account(id, edSha3Crypto.generateKeypair());
+  }
+
+  public static Account createWithEd25519Sha2(String id) {
```

Comment:
maybe move those creational methods to a class in the `test` module?

Reply:
`Account` is used in `QueryAPI`. I think `debug` module is not in `tests` on purpose, so that library users can reuse the module in debugging.

Reply:
I was talking just about those `static` creational methods though